Packs/Jira/ReleaseNotes/3_2_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Atlassian Jira v3
+
+- Fixed an issue in **fetch-incidents** where getting forms from issues in OnPrem DataCenter failed the fetch.
```

Comment:
```suggestion
- Fixed an issue in **fetch-incidents** failed when getting forms from issues in OnPrem DataCenter.
```

Reply:
changed to 
- Fixed an issue in **fetch-incidents** failing when getting forms from issues in OnPrem DataCenter.